### PR TITLE
Test output redirect

### PR DESCRIPTION
**Pull request by stephenmichaelf**
Write test log files to Test\TestLogs instead of netstandard1.5 folder.

#1117

### CODE REVIEW

File: src/Microsoft.VisualStudio.Services.Agent/Constants.cs [hunk 1]
```diff
@@ -149,6 +149,8 @@ public static class Path
                 public static readonly string SourceRootMappingDirectory = "SourceRootMapping";
                 public static readonly string SourcesDirectory = "s";
                 public static readonly string TestResultsDirectory = "TestResults";
+                public static readonly string TestDirectory = "Test";
```
Comment (TingluoHuang) [inline src/Microsoft.VisualStudio.Services.Agent/Constants.cs]: normally, we don't add code into common lib if they only used by Test, you can just use hard code string in the Test proj. :)

**Comment by TingluoHuang** [comment]
do you have change figure out why there is a `_diag` folder under the root of that nuget package local cache?

**Comment by stephenmichaelf** [comment]
Not yet, I will check on the _diag folder on Monday.

File: src/Microsoft.VisualStudio.Services.Agent/Constants.cs (hunk 1 above)
Comment (stephenmichaelf) [inline src/Microsoft.VisualStudio.Services.Agent/Constants.cs]: Fixed :)

**Comment by stephenmichaelf** [comment]
The _diag folder in the netstandard1.5 folder is no longer being created. Working on fixing these broken tests.

**Comment by stephenmichaelf** [comment]
@TingluoHuang This is pending the queued Ubuntu14 build but it's ready for code review, I am done with the refactoring.

**Pull request merged**